.ci.yaml

```diff
@@ -57,9 +57,11 @@ platform_properties:
       caches: >-
         [
           {"name":"flutter_cocoapods","path":"cocoapods"},
-          {"name":"old_osx_sdk","path":"osx_sdk"},
+          {"name":"osx_sdk_13a233_13_15","path":"osx_sdk"},
```

Comment:
I'm not sure what `old_osx_sdk` should be renamed to given https://github.com/flutter/flutter/pull/99586 since the names were different between the engine and framework.  `osx_sdk_13a233_13_15` is for Xcode 13a233 + iOS 13 + iOS 15 but I'm open to suggestions here.

Reply:
old_osx_sdk and osx_sdk were used to transition between two versions of xcode without impacting the builders. Now that we need multiple caches updating the name with version string is a better option.